packages/design-system-tailwind-preset/scripts/testUtils.ts

```diff
@@ -15,31 +15,38 @@ import postcss from 'postcss';
 export const getDesignTokenVariables = async (
```

Comment:
Updating this test util to use the workspace version of the design tokens stylesheet. We can also refactor this now that we can access the css locally